src/main/kotlin/me/zeroeightsix/kami/target/CategorisedTargets.kt

```diff
@@ -174,6 +197,65 @@ enum class ItemCategory(override val belongsFunc: (Item) -> Boolean) : Categoris
             Items.POWERED_RAIL,
             Items.TNT_MINECART
         )
+
+        private val resources = setOf(
+            // normal
+            Items.IRON_INGOT,
+            Items.GOLD_INGOT,
+            Items.NETHERITE_INGOT,
+            Items.DIAMOND,
+            Items.REDSTONE,
+            Items.COAL,
+            Items.CHARCOAL,
+            Items.NETHERITE_SCRAP,
+            Items.LAPIS_LAZULI,
+
+            // blocks
+            Items.IRON_BLOCK,
+            Items.GOLD_BLOCK,
+            Items.NETHERITE_BLOCK,
+            Items.DIAMOND_BLOCK,
+            Items.REDSTONE_BLOCK,
+            Items.COAL_BLOCK,
+            // Items.CHARCOAL_BLOCK, oh wait that doesnt exist
+            Items.LAPIS_BLOCK,
+
+            // nuggets
+            Items.IRON_NUGGET,
+            Items.GOLD_NUGGET,
+            // Items.NETHERITE_NUGGET, oh wait that doesnt exist either
+        )
+
+        private val mobDrops = setOf(
```

Comment:
This is by far not an exhaustive list, but imo it doesn't really need to be - some items are difficult to categorise as 'specifically a mob drop'.

However, I would like to see added:
* Ender pearl
* Gold nugget
* Raw foods: raw beef, raw chicken, raw cod, raw salmon, raw porkchop, raw rabbit, raw mutton

Reply:
do you think i should remove slime balls from the list? for some reason my brain does not categorize them as mob drop but as redstone crafting componenent

Reply:
I'd prefer for slime balls to stay.

Reply:
So you think slime balls are for redstone and redstone is a mob drop?

Reply:
> 
> 
> So you think slime balls are for redstone and redstone is a mob drop?

its just a subjective thing. ill leave them in the list